css/fieldmanager-group-tabs.css

```diff
@@ -89,13 +116,14 @@
 
 .fm-wrapper .wp-tabs-panel > .fm-wrapper .fm-group-inner {
 	border-top: solid 1px #dfdfdf;
+	border-right: none;
+	border-bottom: none;
+	border-left: none;
+	-webkit-box-shadow: none;
```

Comment:
Yes, we'd also like to set up SASS which will allow for a better organization of CSS